### PR TITLE
♻️(project) update docker images build

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

We don't want rely on existing Warren base Docker images to run the project.

## Proposal

- [x] build custom API docker image
- [x] pull APP image from Warren public repository

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -46,7 +46,8 @@ services:
       - "${WARREN_API_SERVER_PORT:-8100}"
       - "--reload"
     volumes:
-      - ./src/api:/app/plugins/tdbp
+      # - ./src/api:/app/plugins/tdbp
+      - ./src/api:/app/
```
Comment (jmaupetit) [inline docker-compose.yml]: Should restore original path.

File: src/api/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,54 @@
...
-# FIXME: use an official release instead
-FROM warren-api:development as base
+FROM fundocker/warren:api-core-main as base
+
+# -- Builder --
+FROM base as builder
+
+WORKDIR /build
+
+# Should be a priviledge user
```
Comment (quitterie-lcs) [inline src/api/Dockerfile]:
```suggestion
# Should be a priviledged user
```

File: src/api/logging-config.dev.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+version: 1
+disable_existing_loggers: false
+formatters:
+  default:
+    "()": uvicorn.logging.DefaultFormatter
```
Comment (quitterie-lcs) [inline src/api/logging-config.dev.yaml:6]: Why `"()"`?

**Review by quitterie-lcs — APPROVED** [review]

File: src/api/logging-config.dev.yaml (hunk 3 above)
Comment (jmaupetit) [reply src/api/logging-config.dev.yaml:6]: This is required for user-defined objects. See: https://docs.python.org/3/library/logging.config.html#user-defined-objects

**Pull request merged**